src/ApiService/ApiService/onefuzzlib/TaskOperations.cs

```diff
@@ -15,7 +15,7 @@ public interface ITaskOperations : IStatefulOrm<Task, TaskState> {
 
     IAsyncEnumerable<Task> SearchStates(Guid? jobId = null, IEnumerable<TaskState>? states = null);
 
-    IEnumerable<Container>? GetInputContainerQueues(TaskConfig config);
```

Comment:
Did you mean to use `OneFuzzResult`? I don't think we use `Result<T, E>` anywhere outside of `FunctionalTests`

Reply:
We should probably just rename `OneFuzzResult` to `Result`…

Reply:
`OneFuzzResult` Error is of OneFuzz Error type `{int code, string[] errors}`

Anywhere in the code where `TaskConfigError` is returned it is returned as `ResultVoid<TaskConfigError>.Error(new TaskConfigError(err));`

In this specific case `ResultVoid` won't do

Next code iteration - maybe might make sense to get rid of `ResultVoid<TaskConfigError>` as well as fix up this code and use `OneFuzzResult` across the board